src/modules/multi-tenancy/tenant.middleware.ts

```diff
@@ -0,0 +1,51 @@
+import { NextFunction, Response } from 'express';
+import { createConnection, EntityManager, getManager, LoggerOptions } from 'typeorm';
+import { getConfigFile } from 'medusa-core-utils/dist';
+import { ShortenedNamingStrategy } from '@medusajs/medusa/dist/utils/naming-strategy';
+import { asValue, AwilixContainer } from 'awilix';
+import TenantRepository from './tenant.repository';
+import { MedusaAuthenticatedRequest, MedusaMiddleware } from '../../core';
+import { Middleware } from '../../decorators';
+import { Tenant } from "./tenant.entity";
```

Comment:
Codacy has a fix for the issue: [Replace `"./tenant.entity"` with `'./tenant.entity'`](https://app.codacy.com/gh/adrien2p/medusa-extender/pullRequest?prid=9061797)
```suggestion
import { Tenant } from './tenant.entity';
```